symengine/sets.cpp

```diff
@@ -90,6 +92,19 @@ RCP<const Set> Interval::close() const
     return interval(start_, end_, false, false);
 }
 
+bool Interval::contains(const RCP<const Basic> &a) const
```

Comment:
This should be a `const RCP<const Number> &` right? Otherwise it will always return false.